assets/js/modules/analytics/dashboard/dashboard-widget-all-traffic.js

```diff
@@ -61,6 +62,30 @@ class AnalyticsAllTraffic extends Component {
 		} );
 	}
 
+	getErrorDataComponent() {
```

Comment:
Double-checking, can't we use the actual logic from `withData` here instead of having to replicate it?

Reply:
@felixarntz I believe we need implement same pattern as Search Funnel that makes use of `<DashboardSearchFunnelInner />` where modules hook into for adding each component, would be better to open another issue for this. What do you think?

Reply:
It's okay for now I think. This will be revisited as part of the JS refactoring.